packages/example-api/src/app/user/user.entity.ts

```diff
@@ -9,11 +9,11 @@ export class User
   implements IMetamaskUserEntity, IGoogleAccountUser, IUsernameAndPassword
 {
   id: number;
-  name: string;
-  googleId: string;
-  email: string;
+  name: string | number;
+  username: IUsernameAndPassword['username'];
```

Comment:
types by default de interface. Why IUsernameAndPassword['username'] ??

Reply:
ywah, you could use string  here or the same type you use in username for IUSernameAndPassword

Reply:
The objective of this was to keep the same type declared in the strategy, so in case of having to change the type it would only be done at a single point. But I made the change for primitive type if it works better for you